src/Symfony/Component/Console/Tests/Helper/SymfonyQuestionHelperTest.php

```diff
@@ -101,6 +103,15 @@ public function testAskEscapeLabel()
         $this->assertOutputContains('Do you want a \?', $output);
```

Comment:
The initial issue was about having a backslash as default value which has been fixed, not really about a question ending with a `\`, which is indeed a very edge case to me. I think this is a good consensus, don't you?